### DISCUSSION TITLE
Developer Portal - Scripts.V3

### DISCUSSION THREAD

**Discussion by Murali1689** [Azure/api-management-developer-portal#1394] [General]
Is it possible to execute node scripts (for e.g.migrate.js or migrate.bat) via the Azure DevOps pipeline? scripts have a prerequisite to complete "Az login" before executing them which is fine in dev machine, how to achieve the same in the Azure DevOps pipeline

**Comment by mikebudzynski** [comment]
There's an open PR for the Azure DevOps integration: https://github.com/Azure/api-management-developer-portal/pull/1401

**Comment by krishnarajsp** [comment]
is this integration with azure devops completed? If yes could you please share the documentation on how to use this in pipeline?
